launcher/res/DashFactionLauncher.manifest

```diff
@@ -1,20 +1,26 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<assembly xmlns="urn:schemas-microsoft-com:asm.v1" manifestVersion="1.0">
+<assembly xmlns="urn:schemas-microsoft-com:asm.v1" manifestVersion="1.0"
+  xmlns:asmv3="urn:schemas-microsoft-com:asm.v3">
   <assemblyIdentity
     version="1.0.0.0"
-    processorArchitecture="X86"
-    name="rafalh.DashFaction.Launcher"
-    type="win32" />
-  <description>DashFaction launcher</description>
+    processorArchitecture="x86"
+    name="rafalh.DashFactionLauncher"
+    type="win32"/>
+  <description>Dash Faction Launcher</description>
   <dependency>
     <dependentAssembly>
       <assemblyIdentity
         type="win32"
         name="Microsoft.Windows.Common-Controls"
         version="6.0.0.0"
-        processorArchitecture="X86"
+        processorArchitecture="x86"
         publicKeyToken="6595b64144ccf1df"
-        language="*" />
+        language="*"/>
     </dependentAssembly>
   </dependency>
+  <asmv3:application>
+    <asmv3:windowsSettings xmlns="http://schemas.microsoft.com/SMI/2016/WindowsSettings">
+      <dpiAwareness>permonitorv2</dpiAwareness> <!-- Requires Windows 10, version 1703 or later. !-->
```

Comment:
This will probably break Dash for old builds of Windows 10 (those that understand `dpiAwareness` tag but don't support `permonitorv2`. There should be some fallback options (you can list many dpiAwareness modes here and the first one supported is used from what I know). If we want to support DPI awareness on older systems something like this would be necessary: https://gist.github.com/emoacht/7e5a026080aeb7eb1b9316f5fe7628da But if enabling `PerMonitor` V1 or `true/PM` would make the launcher window small on old systems (assuming dialogs are not scaled automatically in V1) then we may want make `unaware` the default. It would be best if you could test it in Windows 7/8 with different DPI settings

Reply:
I don't think that it's meaningful to support versions of Windows that are older than 4 years.   I also don't think that permonitorv2 should be opt-in, if you're running a recent version of Windows.   The whole point is so that a user does not have to right-click an exe to open its properties, so that they can change its DPI settings.

Reply:
At this moment DF supports Windows 7+ and I want it to stay that way for some time unless supporting it would be difficult. In this case supporting old systems is like changing one or two lines so please do it. Windows 10 has currently about 78% of usage world-wide (https://gs.statcounter.com/os-version-market-share/windows/desktop/worldwide). This means not supporting older systems makes ~20% of people unable to use Dash Faction which is something I'd like to avoid. I wonder how would look the usage diagram aggregated by Windows 10 patch versions. Perhaps you propose to support systems used by less than 70% of PC users.